docs/process_wrapper_doc.md

```diff
@@ -0,0 +1,130 @@
+## process_wrapper
+
+Process wrapper is a helper that allows you, in a platform independent way,
```

Comment:
I think we should make it clear that using this means you'll have C++ toolchain in your dependency.

Reply:
Added to the doc